src/fal/planner/executor.py

```diff
@@ -27,7 +34,10 @@ def _collect_nodes(groups: List[TaskGroup], fal_dbt: FalDbt) -> Iterator[str]:
     for group in groups:
         if isinstance(group.task, DBTTask):
             yield from group.task.model_ids
-        if isinstance(group.task, FalLocalHookTask) and not group.task.is_hook:
+        if (
+            isinstance(group.task, FalLocalHookTask)
+            and not group.task.hook_type is HookType.HOOK
```

Comment:
From the changes in `_run_hook.py`, couldn't it happen that:

```py
isinstance(group.task, FalLocalHookTask) is True
and group.task.hook_type is HookType.MODEL_SCRIPT
```

Making this condition check `True` and we are assuming they are before/after scripts?

Reply:
Good catch, it makes sense to filters this to only include scripts.